run.sh

```diff
@@ -3,14 +3,17 @@
 checkPatterns() {
     keepit=$3
     if [ -n "$1" ]; then
-        for PATTERN in $(echo $1 | tr "," "\n"); do
-        if [[ "$2" = $PATTERN* ]]; then
-            if [ $DEBUG ]; then echo "DEBUG: Matches $PATTERN - keeping"; fi
-            keepit=1
+      for PATTERN in $(echo $1 | tr "," "\n"); do
+        if [[ $PATTERN = '.' ]]; then
```

Comment:
I don't understand how (or if) the old code here worked. Are you fixing a bug or adding new functionality?